src/AutoLineJob.sol

```diff
@@ -56,8 +56,26 @@ contract AutoLineJob is IJob {
         tlo = _tlo;
     }
 
-    function getNextJob(bytes32 network) external view override returns (bool, address, bytes memory) {
-        if (!sequencer.isMaster(network)) return (false, address(0), bytes("Network is not master"));
+    function execute(bytes32 network, bytes calldata execPayload) external override {
+        require(sequencer.isMaster(network));
```

Comment:
custom errors could come handy for debugging the simulations 🙏 
since they're not supposed to span i believe they would represent no cost at all

Reply:
yes good point